mitosheet/mitosheet/tests/step_performers/graph_steps/test_graph.py

```diff
@@ -83,16 +96,30 @@ def test_all_styling_options():
         False, 
         ['A'], ['B'], 
         400, 400, 
+        color,
+        facet_col_column_id=facet_col_column_id,
+        facet_row_column_id=facet_row_column_id,
+        facet_col_wrap=facet_col_wrap,
+        facet_col_spacing=facet_col_spacing,
+        facet_row_spacing=facet_row_spacing,
         title_title=title_title,
         title_visible=title_visible,
         xaxis_title=xaxis_title,
         xaxis_visible=xaxis_visible,
         xaxis_type=xaxis_type,
+        xaxis_showgrid=xaxis_showgrid,
+        xaxis_gridwidth=xaxis_gridwidth,
         xaxis_rangeslider_visible=xaxis_rangeslider_visible,
         yaxis_title=yaxis_title,
-        yaxis_type=yaxis_type,
         yaxis_visible=yaxis_visible,
+        yaxis_type=yaxis_type,
+        yaxis_showgrid=yaxis_showgrid,
+        yaxis_gridwidth=yaxis_gridwidth,
         showlegend=showlegend,
+        legend_title_text=legend_title_text,
+        legend_orientation=legend_orientation,
+        legend_x=legend_x,
+        legend_y=legend_y,
```

Comment:
It doesn't look like you're testing everything here. Can you check histnorm, for example? And all the rest of them, if they aren't here.

Reply:
I don't test the graph params, like histnorm, that are only for specific graph types in that parameritized function because it will error when trying to apply the histnorm param to other graph types. 

Since we moved to the model of trusting the parameters from the frontend, we don't do a second round of testing on the backend to ensure that the params only get applied to the correct graph types. 

Instead, I created a second set of tests directly below this one that test the specific graph type params.